torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -178,7 +178,9 @@ C10_REGISTER_CREATOR(
     mpt_uv,
     makeMultiplexedUvChannel);
 
-#if TENSORPIPE_USE_CUDA_IPC_CHANNEL
+#if TENSORPIPE_HAS_CUDA_IPC_CHANNEL && defined(USE_CUDA_NOT_ROCM)
+
+constexpr int64_t kCudaIpcChannelPriority = 300;
```

Comment:
Nit: I agree that this constant is only needed when the CUDA IPC channel is enabled, and thus it could be guarded inside the `#if`, but I still think it would be nice to have all the priorities grouped together since it's useful to have an overview and compare them.